api/src/services/__tests__/programsService.ts

```diff
@@ -611,4 +623,26 @@ describe('programsService', () => {
       });
     });
   });
+
+  describe('listParticipantActivitiesCompletionForProgram', () => {
+    const programId = 1;
+    const principalId = 3;
```

Comment:
These two `const` declarations are done outside of the test. I would recommend moving them inside of the test unless these values are shared by multiple tests.